### DISCUSSION TITLE
Rainbow show rainbow indent after editing code

### DISCUSSION THREAD

**Discussion by LKAMinco** [izhangzhihao/intellij-rainbow-brackets#2680] [General]
Is is possible to have always shown rainbow indent ? Because after editing the code, it gray out, it it is not colorfull anymore.
- before
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/70859762/8fc0cd2f-1498-4b28-87d8-2801a5759096)
- after edit
![image](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/70859762/01c41b91-3f39-4ee4-9d83-d6bee88ce751)

**Comment by izhangzhihao** [comment]
Related to https://github.com/izhangzhihao/intellij-rainbow-brackets/issues/2673

**Comment by izhangzhihao** [comment]
Available in the next release.
